### ISSUE TITLE
Only compare packages exported by modules 

### ISSUE THREAD

**Issue by AlanBateman** [marchof/java-almanac#179]
Looking at this page:
https://javaalmanac.io/jdk/22/apidiff/21/

where it shows a new method added to java.awt.peer.WindowPeer. This is confusing to see because this is an internal class, the java.awt.peer package is not an exported/documented/supported interface. The diff tool shouldn't be looking at APIs in non-exported packages.

**Comment by marchof** [comment]
You're right. Currently I simply filter by package name. It's on my todo list since quite some time to consider the exports in module-info.class.

**Comment by marchof** [comment]
I run a prototype: `java.awt.peer` is the only instance where a `java.*` package is not exported in the JDK.

**Comment by AlanBateman** [comment]
> I run a prototype: java.awt.peer is the only instance where a java.* package is not exported in the JDK.

That's right, the peers were developed in this package a long time ago but never part of the documented/supported API. When the platform moved to modules in Java 9, the java.awt.peer package was not exported.

**Comment by marchof** [comment]
Fixed with ffdb46f53010814a0a29740887861b3bb0711a0f
